webapp/src/shared/components/Row/ScoreRow.css

```
@@ -5,35 +5,54 @@
   grid-template-rows: 1fr;
   width: 100%;
   min-width: fit-content;
-  padding: 0.5rem 1.75rem;
+  padding: 0.5rem 0.5rem;
```

```suggestion
  padding: 0.5rem 1rem;
```

suggestion: Currently, there is a horizontal scrollbar on the score list. This change seems to fix it.

---

You're right thanks